dist/index.js

```diff
@@ -55,7 +61,7 @@ function calculateHistogram(arr) {
     dataCopy.forEach(function (item) {
         var binIndex = Math.floor((item - min) / binSize);
         // for values that lie exactly on last bin we need to subtract one
-        if (binIndex === numBins) {
+        if (binIndex === numBins || JSON.stringify(binIndex) === numBins) {
```

Comment:
@fdpamb I'm just getting around the updating this module. Thanks for the suggestions. I don't think I will merge this change, as I consider it to be responsibility of the module to handle parameters in multiple types. The user can easily convert this a number prior to calling the function